openquake/calculators/event_based_risk.py

```diff
@@ -200,13 +209,12 @@ class EventBasedRiskCalculator(base.RiskCalculator):
     core_func = event_based_risk
 
     epsilon_matrix = datastore.persistent_attribute('epsilon_matrix')
-    spec_indices = datastore.persistent_attribute('spec_indices')
     is_stochastic = True
 
     outs = collections.OrderedDict(
         [('AVGLOSS', 'avg_losses-rlzs'),
-         ('AGGLOSS', 'agg_losses-rlzs'),
-         ('SPECLOSS', 'specific-losses-rlzs'),
+         ('AGGLOSS', 'agg_losses'),
```

Comment:
The name `agg_losses` is consistent with the name used in scenario_risk. Also, `agg_losses` is implemented as a simple array, whereas `asset_loss_table` is a sequence of records, i.e. a "table" in my mind. Should we change the implementation to use a sequence of records also for the `agg_losses` we could change the name. Notice that the name visible to the user after a run of the engine (the `display_name`) can be anything we want.